README.md

```diff
@@ -16,124 +16,111 @@ npm install bedrock-mail
 ```
 
 ```js
-var bedrock = require('bedrock');
-var brMail = require('bedrock-mail');
-
-// OPTIONAL: add triggers in your module init function
-brMail.registerTrigger('getValue', function(event, callback) {
-  // custom code run for trigger
-  // for example, get a value for the email event
-  myLib.getValue(
-    null, event.details.valueId, function(err, value) {
-    if(!err) {
-      event.details.value = value;
-    }
-    callback(err);
-  });
-});
+const bedrock = require('bedrock');
+const brMail = require('bedrock-mail');
+const {config} = bedrock;
+
+// configure default mail behavior
+// defaults to send if NODE_ENV is 'production'
+//config.mail.send = true;
+// defaults for every message
+config.mail.message = {
+  from: 'My Company Support <support@example.com>'
+};
 
-// setup events
-bedrock.config.mail.events.push({
-  type: 'myModule.myEvent',
-  // email for admin
-  template: 'myModule.myEvent'
-}, ...);
-var ids = [
-  'myModule.myEvent',
-  ...
-];
-ids.forEach(function(id) {
-  config.mail.templates.config[id] = {
-    // adjust to point to your module email templates
-    filename: path.join(__dirname, '..', 'email-templates', id + '.tpl')
-  };
-});
-// can set values for use in all events
-bedrock.config.mail.vars.service = {
-  name: 'My Service'
+// setup SMTP transport
+bedrock.config.mail.transport = {
+  type: 'smtp',
+  options: {
+    host: 'mail.example.com'
+  },
+  verify: true
 };
-bedrock.config.mail.vars.support = {
-  name: 'My Service'
-  email: 'support@example.com'
+
+// setup default template locals for all mails
```

Comment:
What does "locals" mean?

Reply:
It's the template data container terminology used by https://email-templates.js.org/.  We had "mail.vars" before, which aligned with "view.vars", which has since been dropped.  I always thought "vars" was also just as confusing.  What would you suggest instead?

Reply:
I think `locals` is fine as the `locals` are merged with a template which becomes a full fleshed out HTML email remotely. We might want to add a note or a local to their docs:

`@see https://email-templates.js.org/#/`

or something as when I think of local I think of the local file system first. So a disambiguation might work.

Reply:
The docs already link to and say it's using email-templates.  Their docs describe how the params like message and locals work.  I had hoped their docs in addition to the simple example would be enough.  I think someone with new eyes on the docs needs to suggest a PR with improved text.  I'd rather not duplicate their docs if possible.